src/pages/board/[slug].tsx

```diff
@@ -57,6 +58,18 @@ export const getServerSideProps: GetServerSideProps<BoardGetServerSideProps> = a
   const config = await getFrontendConfig(routeParams.data.slug);
   const translations = await getServerSideTranslations(boardNamespaces, locale, req, res);
 
+  const session = await getServerAuthSession({ req, res });
```

Comment:
Maybe we could only get the session when the guest access is restricted. Then we make a call less to the backend.